webapp/src/pages/EditUserPage/EditUserPage.css

```diff
@@ -0,0 +1,58 @@
+.edit-user-page {
+  margin: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  height: 100vh;
+  background-color: var(--color-background);
+  color: var(--color-light);
+  display: grid;
+  gap: 4rem var(--column-gap);
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto 1fr auto;
+  overflow: scroll;
+}
+
+.edit-user-form {
+  min-width: 15rem;
+  max-width: 20rem;
+  justify-self: center;
+}
+
+.edit-user-form-button {
+  justify-self: center;
+  align-self: end;
+}
+
+@media all and (min-width: 768px) {
+  .edit-user-page {
+    display: grid;
+    grid-template-columns: repeat(var(--column-number), 1fr);
+    grid-template-rows: auto 1fr auto;
+    grid-row-gap: 8rem;
+  }
+  .edit-user-page .header {
+    grid-row: 1;
+    grid-column: 1 / var(--column-end);
+  }
+
+  .avatar-large {
+    grid-column: 2/6;
+    grid-row: 2/3;
+    align-self: start;
+  }
+  .edit-user-form {
+    grid-column: 7/12;
+  }
+  .edit-user-form-button {
+    grid-column: 5/9;
+  }
+}
+
+@media (min-width: 1024px) {
```

Comment:
we kind of agreed that the bigger version would start at 1200px... do you mind?

Reply:
No problem. Done!